oceannavigator/frontend/src/components/Map.jsx

```diff
@@ -1260,17 +1266,18 @@ export default class Map extends React.PureComponent {
       // Disable zooming when drawing
       this.controlDoubleClickZoom(false);
       const lonlat = olproj.transform(e.feature.getGeometry().getCoordinates(), this.props.state.projection, "EPSG:4326");
+      let lonlat_vector = [...this.state.lonlat, [lonlat[1], lonlat[0]]]
```

Comment:
If a variable is not being modified, always declare it `const`.
```suggestion
      const drawn_lonlats = [...this.state.lonlat, [lonlat[1], lonlat[0]] ];
```
FYI JS does not have a vector concept; only arrays.